cli/src/cmd_net.rs

```diff
@@ -325,7 +325,7 @@ impl AuthenticatedCmd for CmdBgpWithdraw {
     async fn run(&self, client: &Client) -> Result<()> {
```

Comment:
line 309: "address lot" should be "announce set", there is no address lot specified here.

Test (continuation of test for line 250, use the resulting state from there for initial state here):

Invoke command:
```
$ oxide system networking address-lot block list --address-lot cloud-pool2
[
  {
    "first_address": "203.0.153.1",
    "id": "f75b71ae-5e4f-4c51-b37c-182cae3a54f1",
    "last_address": "203.0.153.254"
  }
]
```

Resulting state:
```
$ oxide system networking bgp announcement list --announce-set as65547-announce
[
  {
    "address_lot_block_id": "792720c3-373d-4b01-b854-2f7c19402467",
    "announce_set_id": "387b0ed7-5aa4-4017-9c88-7343c1285605",
    "network": "198.51.100.0/24"
  },
  {
    "address_lot_block_id": "792720c3-373d-4b01-b854-2f7c19402467",
    "announce_set_id": "387b0ed7-5aa4-4017-9c88-7343c1285605",
    "network": "203.0.113.0/24"
  }
]
```

Reply:
For clarity, here is the help output:
```
$ oxide system networking bgp withdraw --help
Withdraw a prefix over BGP.

This command removes the provided prefix to the specified announce set.
The remove is performed as a read-modify-write on the specified address lot.

Usage: oxide system networking bgp withdraw [OPTIONS] --announce-set <ANNOUNCE_SET> --prefix <PREFIX>

Options:
      --announce-set <ANNOUNCE_SET>
          The announce set to withdraw from

      --prefix <PREFIX>
          The prefix to withdraw

  -h, --help
          Print help (see a summary with '-h')

Global Options:
      --profile <PROFILE>
          Configuration profile to use for commands
```